tomcat/build.go

```diff
@@ -217,3 +223,17 @@ func (b Build) tinyStartCommand(homePath, basePath string, loggingDep libpak.Bui
 
 	return command, arguments
 }
+
+func (b Build) containsWarFiles(dir string) (bool, error) {
+	files, err := ioutil.ReadDir(dir)
+	if err != nil {
+		return false, err
+	}
+
+	for _, file := range files {
+		if strings.HasSuffix(file.Name(), ".war") {
+			return true, nil
+		}
+	}
+	return false, nil
+}
```

Comment:
You can move this to a central file to share.

 In Go, you can have an `internal` package (so make a new folder called `internal`) and inside that you can put code that is only accessible to this project but you can still export it as public. So you could have this method be `ContainsWarFiles(dir string) (bool, error)` and it would be accessible from any part of the app, but not outside of the project.

Reply:
Also, don't use ioutil, like mentioned in another comment.

Reply:
@dmikusa, thanks. I moved it to internal package `util`.